tests/zfs-tests/tests/functional/slog/slog_replay_fs.ksh

```diff
@@ -175,7 +190,9 @@ log_must cp -a /$TESTPOOL/$TESTFS/* $TESTDIR/copy/
 log_must zfs unmount /$TESTPOOL/$TESTFS
 
 log_note "Verify transactions to replay:"
-log_must zdb -iv $TESTPOOL/$TESTFS
+# Use -ddd to warmup cache for faster replay to make sure TX_REMOVE and
+# TX_CREATE will happen in same txg
```

Comment:
I'm not sure I understand how adding `-ddd` helps warmup the cache.  Which cache?

Reply:
I'm not exactly sure, but I could only reproduce it with `-ddd`. It could be that I'm testing it on VM.

Reply:
I moved the test to a separate test case and it doesn't require `-ddd` to reproduce.